File: ComiAPI/ComiCore/Model/Director.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ComiCore.Model
{
    public class Director: BaseEntity
    {
        public string Name { get; set; }
        public string Photo { get; set; }
        public virtual ICollection<MoviesDirector> MoviesDirectors { get; set; }
    }
}

File: ComiMovies/src/app/models/director.ts
import { BaseEntity } from './ientity';

export class Director extends BaseEntity {

    constructor(id:number, name:string, photo:string){
        super();
        this.id = id;
        this.name = name;
        this.photo = photo;
    }

    name: string
    photo: string
    moviesDirectors: Array<any>
}
